### ISSUE TITLE
Code issue in `modifyvariables.dm`

### ISSUE THREAD

**Issue by amylizzle** [goonstation/goonstation#12734]
### Labels

[BUG]

### Description

Bad use of initial

### Steps to reproduce

`Warning OD2206 at code\modules\admin\viewvariables\modifyvariables.dm:106:41: calling initial() on a local variable returns the current value`

There's no way this could possibly work as intended, it'll probably need a rewrite.

### Expected Behavior

Don't use initial wrong

### Additional Information & Screenshots

OD compiler test

**Comment by frawhst** [comment]
I don't think the intent actually works at all? I added that case because it was available in the legacy input system, but since lists are all null to `initial`, resetting a list element to its initial value is essentially meaningless. Even if `initial` didn't work, how would you distinguish list elements from each other? If I removed a few elements at random places, how would the remaining ones know what their initial value was?

tldr I think I can just delete the case wholesale

**Comment by frawhst** [comment]
Fixed by pali in 59cf5c1a7623a8a0439abd87849f40f1e52875c3
